File: 백준/Silver/11723. 집합/집합.java
import java.io.*;
import java.util.HashSet;
import java.util.StringTokenizer;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
        StringBuilder sb = new StringBuilder();
        HashSet<Integer> set = new HashSet<>();

        int n = Integer.parseInt(br.readLine());

        for (int i = 0; i < n; i++) {
            StringTokenizer st = new StringTokenizer(br.readLine());

            String s = st.nextToken();
            int x;

            if (s.equals("add")) {
                x = Integer.parseInt(st.nextToken());
                set.add(x);
            } else if (s.equals("remove")) {
                x = Integer.parseInt(st.nextToken());
                set.remove(x);
            } else if (s.equals("check")) {
                x = Integer.parseInt(st.nextToken());
                if (set.contains(x)) {
                    sb.append(1).append("\n");
                } else {
                    sb.append(0).append("\n");
                }
            } else if (s.equals("toggle")) {
                x = Integer.parseInt(st.nextToken());
                if (!set.contains(x)) {
                    set.add(x);
                } else {
                    set.remove(x);
                }
            } else if (s.equals("all")) {
                set = new HashSet<>();
                for (int j = 1; j <= 20; j++) {
                    set.add(j);
                }
            } else if (s.equals("empty")) {
                set.clear();
            }
        }
        bw.write(sb.toString());
        bw.flush();
        bw.close();
    }
}

File: 백준/Silver/11723. 집합/집합.py
import sys

m = int(sys.stdin.readline())
s = set()

for i in range(m):
    command = sys.stdin.readline().split()
    if command[0] == 'add':
        s.add(int(command[1]))
    elif command[0] == 'remove':
        s.discard(int(command[1]))
    elif command[0] == 'check':
        if int(command[1]) in s:
            print(1)
        else:
            print(0)
    elif command[0] == 'toggle':
        if int(command[1]) in s:
            s.remove(int(command[1]))
        else:
            s.add(int(command[1]))
    elif command[0] == 'empty':
        s.clear()
    elif command[0] == 'all':
        s = set(range(1, 21))